README.md

```diff
@@ -1,2 +1,25 @@
-# tsdbcg
-A Deck Building Card Game built using X and Y working with A to Z.
+## POST `api/v1/games/:id/buy_card`
+This endpont allows a player in a game to but a card for that case. A POST with the body
```

Comment:
I believe this should read, "This endpoint allows a Player in a Game to buy a Card during their turn. A required Body follows the format below:"